src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -149,8 +149,10 @@ export class ESLToggleable extends ESLBaseElement {
     if (!this.connected || newVal === oldVal) return;
     switch (attrName) {
       case 'open':
-        if (this.open === this.hasAttribute('open')) return;
-        this.toggle(this.open, {initiator: 'attribute', showDelay: 0, hideDelay: 0});
+        // eslint-disable-next-line no-case-declarations
+        const isOpen = this.hasAttribute('open');
+        if (this.open === isOpen) return;
+        this.toggle(isOpen, {initiator: 'attribute', showDelay: 0, hideDelay: 0});
```

Comment:
``` suggestion
      case 'open': {
        const isOpen = this.hasAttribute('open');
        if (this.open === isOpen) return;
        this.toggle(isOpen, {initiator: 'attribute', showDelay: 0, hideDelay: 0});
      }
```
Please don't disable the linting rules. 
In case lint have something to say, most likely we need to fix it, otherwise, we need to discus disabling globally. There are only a few moments when we disable some rules locally.